melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/fra_sed/sed_grunnlag/SedGrunnlagMapperA003.java

```diff
@@ -0,0 +1,86 @@
+package no.nav.melosys.eessi.service.sed.mapper.fra_sed.sed_grunnlag;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
```

Comment:
What is dis. Tipper den blir dratt inn gjennom token-support biblioteket? Jeg ville kanskje heller brukt !isEmpty istedenfor å ta i bruk denne 😄

Reply:
Oisann, det la jeg ikke merke til! 😅